awsbestpractices/mandatory-method/MandatoryMethod.java

```diff
@@ -0,0 +1,31 @@
+import javax.swing.ImageIcon;
+import com.amazonaws.services.simplesystemsmanagement.model.PutParameterRequest;
+
+public class MandatoryMethod {
+
+    public void checkWindowDescription() {
+        //Should not flag
+        CreateMaintenanceWindowRequest maintainanceWindow = new CreateMaintenanceWindowRequest();
+        maintainanceWindow.setDescription("Description");
+
+        //Should not flag
+        CreateMaintenanceWindowRequest maintainanceWindow2 = new CreateMaintenanceWindowRequest();
+        maintainanceWindow2.withDescription("Description");
+
+        //Should flagS
+        CreateMaintenanceWindowRequest maintainanceWindow3 = new CreateMaintenanceWindowRequest();
+        System.out.println(maintainanceWindow3);
+
+    }
+    public void checkParameterDescription() {
+
+        //Should not flag
+        PutParameterRequest putParameterRequest = new PutParameterRequest();
+        putParameterRequest.setDescription("Description");
+
+        //Should flag
+        PutParameterRequest putParameterRequest2 = new PutParameterRequest();
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Add information about the parameter that you want to add to the system by calling [setDescription()](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/simplesystemsmanagement/model/PutParameterRequest.html#setDescription-java.lang.String-) method. Optional but recommended.